django/test/testcases.py

```diff
@@ -145,7 +145,7 @@ def __init__(self, wrapped, message):
         self.message = message
 
     def __call__(self):
-        raise AssertionError(self.message)
+        raise DatabaseFailureError(self.message)
```

Comment:
`DatabaseFailureError` is not declared, it should be a subclass of `AssertionError`.

Reply:
working on it :)

Reply:
should I declare a subclass of `AssertionError` in `django/core/exceptions.py`?